lib/ansible/modules/system/seboolean.py

```diff
@@ -49,7 +49,7 @@
     choices: [ 'yes', 'no' ]
 notes:
    - Not tested on any debian based system
-requirements: [ ]
+requirements: [ 'libselinux-python', 'libsemanage-python' ]
```

Comment:
It looks like seboolean was first committed in version 0.7 97c2e58081c9c7c139916d8164bd649ba8409c0e, and it has always had these requirements, but when the documentation was added later in 054f99b10de6c092232855b1e6a27d0160020059, it did not include them. What is the proper way to indicate then that version 0.7 is the version these requirements were added? Would that go in the `notes` part?
http://docs.ansible.com/ansible/dev_guide/developing_modules_documenting.html#documentation-block only talks about optionally specifying the minimum version required.